packages/evolution-common/src/services/baseObjects/BasePerson.ts

```diff
@@ -274,57 +249,6 @@ class BasePerson extends Uuidable implements IBasePersonAttributes, IValidatable
             errors.push(new Error('BasePerson validateParams: educationalAttainment is not a valid value'));
         }
 
-        // Validate baseWorkPlaces (if provided)
-        if (
-            dirtyParams.baseWorkPlaces !== undefined &&
-            (!Array.isArray(dirtyParams.baseWorkPlaces) ||
-                !dirtyParams.baseWorkPlaces.every((place) => place instanceof BasePlace))
-        ) {
-            errors.push(new Error('BasePerson validateParams: baseWorkPlaces should be an array of BasePlace'));
-        }
-
-        // Validate baseSchoolPlaces (if provided)
-        if (
-            dirtyParams.baseSchoolPlaces !== undefined &&
-            (!Array.isArray(dirtyParams.baseSchoolPlaces) ||
-                !dirtyParams.baseSchoolPlaces.every((place) => place instanceof BasePlace))
-        ) {
-            errors.push(new Error('BasePerson validateParams: baseSchoolPlaces should be an array of BasePlace'));
-        }
-
-        // Validate baseHome (if provided)
-        if (dirtyParams.baseHome !== undefined && !(dirtyParams.baseHome instanceof BasePlace)) {
-            errors.push(new Error('BasePerson validateParams: baseHome should be an instance of BasePlace'));
-        }
-
-        // Validate baseVisitedPlaces (if provided)
-        if (
-            dirtyParams.baseVisitedPlaces !== undefined &&
-            (!Array.isArray(dirtyParams.baseVisitedPlaces) ||
-                !dirtyParams.baseVisitedPlaces.every((place) => place instanceof BaseVisitedPlace))
-        ) {
-            errors.push(
-                new Error('BasePerson validateParams: baseVisitedPlaces should be an array of BaseVisitedPlace')
-            );
-        }
-
-        // Validate baseTrips (if provided)
-        if (
-            dirtyParams.baseTrips !== undefined &&
-            (!Array.isArray(dirtyParams.baseTrips) || !dirtyParams.baseTrips.every((trip) => trip instanceof BaseTrip))
-        ) {
-            errors.push(new Error('BasePerson validateParams: baseTrips should be an array of BaseTrip'));
-        }
-
-        // Validate baseVehicles (if provided)
-        if (
-            dirtyParams.baseVehicles !== undefined &&
```

Comment:
Where are those validations now?

Reply:
they were never used, this was base object composition, which should not happen. We will composed only custom objects in survey itself.